t/plugin/traffic-split2.t

```diff
@@ -799,3 +799,75 @@ GET /t
 --- error_code: 500
 --- error_log
 failed to find upstream by id: invalid-id
+
+
+
+=== TEST 21: use upstream with https scheme
```

Comment:
The test case can pass without your pr

Reply:
looks like using `upstream_id` supports https but using real upstream is not supported, I have pushed my changes now, please review.